Answers/hw03_conwellddw.py

```diff
@@ -0,0 +1,121 @@
+#################################################################################
+# Author: Dayton Conwell
+# Username:conwelld
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:to draw a unique object
+# Google Doc Link:https://docs.google.com/document/d/16yg3ch82DAR6bArUB5gwlDlm5b43f2yEVE2Y9IcKr5g/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+# Professor Scott Hegen
+#
+#################################################################################
+
+import turtle
+
+wn = turtle.Screen() #initialize screen window
+wn.bgcolor('grey') #sets bg color to gray
+turtle.colormode(255) #allows for rgb values to be input for color
```

Comment:
There should be no code outside of functions now. Move these lines to the beginning of your `main` function.